src/components/LandingPage.jsx

```diff
@@ -80,52 +83,57 @@ const LandingPage = ({history}) => {
                         height={300}
                         margin={{left: 10, bottom: 30}}
                         data={data.monthly}
+                        style={{marginLeft: 'auto', marginRight: 'auto'}}
                     >
-                        <Label value="Monthly DOMINO Executions" position="top"/>
-                        <CartesianGrid strokeDasharray="3 3"/>
+                        <CartesianGrid strokeDasharray="3 3" stroke='white'/>
                         <XAxis
                             dataKey="date"
-                            label={{value: 'Date', position: 'bottom'}}
+                            label={{value: 'Date', position: 'bottom', fill: 'white'}}
+                            tick={{ fill: 'white' }} 
                         ></XAxis>
                         <YAxis
-                            label={{value: 'Frequency', angle: -90, position: 'left'}}
+                            label={{value: 'Frequency', angle: -90, position: 'left', fill: 'white'}}
+                            tick={{ fill: 'white' }}
                         />
                         <Tooltip/>
                         <Bar dataKey="freq" fill="#007bff"/> {/*80bdff*/}
                     </BarChart>
                 </Col>
                 <Col>
                     <div className={circle}>{data.total}</div>
-                    <p style={{fontSize: '22px'}}>{attr} this year</p>
+                    <p style={{fontSize: '22px', width: '180px', textAlign: 'center', color: 'white'}}>{attr} this year</p>
                 </Col>
             </Row>
         );
     };
 
     const getCard = (cardDetail) => {
+
+
         switch (cardDetail) {
             case "developerCredits":
-                return DeveloperCredits;
+                return (<DeveloperCreditsCard cardStatus={details=="developerCredits"}/>);
             case "researchGroups":
-                return ResearchGroupCard;
+                return (<ResearchGroupCard cardStatus={details=="researchGroups"}/>);
             case "citation":
-                return CitationCard;
+                return (<CitationCard cardStatus={details=="citation"}/>);
             case "repositories":
-                return RepositoriesCard;
+                return (<RepositoriesCard cardStatus={details=="repositories"}/>);
             case "contact":
-                return ContactAndIssuesCard;
+                return (<ContactAndIssuesCard cardStatus={details=="contact"}/>);
```

Comment:
From what I can tell, we do not need the `cardStatus` prop.

The switch statement only returns a single card. In the case that `details === "repositories"`, for example, we will execute that case suite and there's no need to recompute `details === "repositories"` for cardStatus, as it is guaranteed to always be true. If `details` changes it's state, the switch statement will return a different card.

Please let me know if I'm missing anything!

Reply:
Actually this entire function should be deleted. Let's discuss this in our next meeting.